requirements-global.txt

```diff
@@ -34,6 +34,7 @@ lion-pytorch==0.2.2 # lion optimizer
 prodigyopt==1.0 # prodigy optimizer
 schedulefree==1.3.0 # schedule-free optimizers
 pytorch_optimizer==3.3.0 # pytorch optimizers
+prodigy-plus-schedule-free==1.8.0
```

Comment:
1.9.0 got released rather recently. Only interface change is an extra parameter, `factored_fp32` with a default of True.